### DISCUSSION TITLE
who can tell me what is sub slot?

### DISCUSSION THREAD

**Discussion by xnzhao196** [Chia-Network/chia-blockchain#2260] [Q&A]
im reading source code of chia ,but i dont understand sub_slot、sub_slot_iters 、 signage_point ?
Who can help me explain?
Where can I study in detail ？

**Comment by hoffmang9** [comment]
https://docs.google.com/document/d/1tmRIb7lgi4QfKkNaxuKOBHRmwbVlGL4f7EsBDr_5xZE/edit
